lib/matplotlib/backends/backend_wx.py

```diff
@@ -619,7 +619,6 @@ def gui_repaint(self, drawDC=None):
         # DC (see GraphicsContextWx._cache).
         bmp = (self.bitmap.ConvertToImage().ConvertToBitmap()
                if wx.Platform == '__WXMSW__'
-                  and isinstance(self.figure._cachedRenderer, RendererWx)
```

Comment:
This one may be implicitly gating that the first expression is going to work or not.